tests/middleware.test.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { devtools, combine } from '../src/middleware'
+import create from '../src/index'
+
+describe('devtools', () => {
+  it('create store with devtools', async () => {
+    // assert redux devtools
+    ;(window as any).__REDUX_DEVTOOLS_EXTENSION__ = {
+      connect(obj: any) {
+        expect(obj.name).toBeUndefined()
+
+        return {
+          subscribe() {},
+          init(initialState: any) {
+            expect(initialState.count).toBe(0)
+            expect(typeof initialState.inc).toBe('function')
+          },
+          send(action: string, payload: any) {
+            expect(action).toBe('action')
+            expect(payload.count).toBe(1)
+            expect(typeof payload.inc).toBe('function')
+          },
+        }
+      },
+    }
+
+    const useStore = create<any>(
```

Comment:
I guess one of our goals is to type correctly (cf. #216)
Can we remove `<any>` for `create()` or type it properly?

Reply:
sure